src/teaching-element/Assessment/DragDrop.vue

```diff
@@ -99,12 +100,15 @@ export default {
     },
     isBoxFull() {
       return this.answersCollection.length === 0;
+    },
+    removeClass() {
+      return get(this.options.dragDrop, 'removeClass', 'mdi mdi-close');
```

Comment:
Kind reminder that naming things properly is hard...
There are two properties attached on every DOM element out there
1. [`Element.className`](https://mdn.io/classname)
2. [`Element.classList`](https://mdn.io/classlist)

Judging by your defaults this should be called `removeClassName` instead :bowtie: